### PR TITLE
fix: make lifecycle methods part of inherited class in federation services

### PR DESCRIPTION
**Pull request by debdutdeb**
- fix: make lifecycle methods part of inherited class in federation services
- chore: add changeset

### CODE REVIEW

**Comment by debdutdeb** [comment]
@MarcosSpessatto quickest one i could think, maybe we could just move them without calling super? i didn't see them used anywhere else but just to be sure.

File: apps/meteor/server/services/federation/service.ts [hunk 1]
```diff
@@ -330,4 +330,12 @@ export class FederationService extends AbstractBaseFederationService implements
 		await federationService.initialize();
 		return federationService;
 	}
+
+	public async stopped(): Promise<void> {
+		return super.stopped();
+	}
+
+	public async created(): Promise<void> {
+		return super.created()
```
Comment (debdutdeb) [inline apps/meteor/server/services/federation/service.ts:339]:
```suggestion
		return super.created();
```

**Comment by MarcosSpessatto** [comment]
> @MarcosSpessatto quickest one i could think, maybe we could just move them without calling super? i didn't see them used anywhere else but just to be sure.

You are right, we can move them, I don't know why I've put this on the base class 🤦‍♂️

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]
Can we follow your suggestion and move them lifecycle methods to the service class instead? I think it would be clearer.

**Comment by debdutdeb** [comment]
actually looks like I can't that easily - 
this
https://github.com/RocketChat/Rocket.Chat/blob/9c06e1d6803c59088c1436dc04f0ae734f0e3772/apps/meteor/ee/server/local-services/federation/service.ts#L138
calls this
https://github.com/RocketChat/Rocket.Chat/blob/9c06e1d6803c59088c1436dc04f0ae734f0e3772/apps/meteor/server/services/federation/service.ts#L300
and is then shared with ce federation service.

**Review by MarcosSpessatto — APPROVED** [review]

**Pull request merged**